app/src/components/Login.tsx

```
@@ -65,93 +65,102 @@ export const Login = () => {
                     {isSignUp && (
-                        <div className="space-y-2 animate-in fade-in slide-in-from-top-1 duration-200">
-                            <label className="text-xs font-bold text-slate-500 uppercase tracking-wider ml-1">
+                        <div className="space-y-2 animate-in fade-in slide-in-from-top-4 duration-500">
+                            <label className="text-[10px] font-bold text-indigo-300 uppercase tracking-[0.2em] ml-1">
                                 User Name
                             </label>
                             <div className="relative group">
-                                <User size={18} className="absolute left-4 top-1/2 -translate-y-1/2 text-slate-400 group-focus-within:text-indigo-500 transition-colors" />
+                                <User size={18} className="absolute left-4 top-1/2 -translate-y-1/2 text-indigo-300/50 group-focus-within:text-indigo-400 transition-colors" />
                                 <input
                                     type="text"
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)